src/Conference/ConferenceDetails.jsx

```diff
@@ -48,15 +54,27 @@ const ConferenceDetails = ({ conference, id, isLoading, error }) => {
     <StyledForm>
       <StyledCard>
         <StyledCardHeading className="name">{name}</StyledCardHeading>
-        <StyledLink to={`/${id}/edit`}>Edit Conference</StyledLink>
+        {sessionToken ? (
```

Comment:
FYI you could write this as `{ sessionToken && ( <YourComponent /> )}` This works as the logical operator(&&) will actually evaluate and show the component if sessionToken exists.

Reply:
didn't realise that was possible - looks much cleaner now, thanks